website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,198 +1,260 @@
-| **updatedRowIndices**        | Binding     | This property contains an array of edited row indices.                                                        | `{{Table1.updatedRowIndices}}` |
-| **updatedRow**               | Binding     | This property contains the details of the row that triggered the action (`onSubmit`, `onSave` or `onDiscard`) | `{{Table1.updatedRow}}`|
-
-#### Update mode
-
-Controls the save experience of an edited cell.
-**Row level** - Cells can be saved using the Save/Discard column buttons\
-**Custom** - Cells can be saved by using an **onSubmit** action of the column or through an external button.
+#### Regex
 
-#### Cell wrapping
+[Regular expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) (regex) are patterns that describe valid user input. For example, the pattern `[a-zA-Z]` describes a string that contains only letters of the English alphabet. Using regular expressions in the **Regex** property of columns, you can set specific constraints on the input you expect from the user.
 
-Controls how overflowing contents of the column are handled. When turned on, the contents get wrapped to the next line.
+When you add a regular expression to a column, all user input in that column is compared to the pattern; it's considered to be valid when it matches the expression, or invalid when it doesn't match. When input is invalid, the cell displays its **Error message** to the user in a tool-tip.
 
-#### Editable
+![](/img/as_errormsg.png)
 
-Controls whether cells of the column are editable. To make the column editable, click the checkbox inside a column card that makes all the cells in that column editable.
+For example, add a regular expression for entering a name. The name can contain only alphabetical characters and spaces between the first and last name:
+
+```
+/^[a-z -]+$/i
+```
 
-Alternatively, A column can be editable by turning on `Editable` property inside column settings.
+If you enter a value that contains something other than alphabetical or space characters, the widget shows an error message "Invalid input."
 
+#### Required
 
-<VideoEmbed host="youtube" videoId="BW5cVp0GfLE" title="Making column editable" caption="Making column editable"/>
+Makes input to the widget mandatory. Sets whether a non-empty value must be entered for the editable cell input.
 
+#### Min
+
+Sets the minimum allowed value. For example, you could set the minimum value to 2 if you only want values greater than 2. Any number entered that's less than 2 is considered invalid.
 
 :::info
-Validation ensures that only certain values are entered in a field. You have to make the column editable to allow validation. The validation feature is available for **text** and **numeric** column types.
+**Min** and **Max** properties are available for **numeric**  column types.
```

Comment:
I tried something slightly different to see how it looked/felt -- I added a sentence to both properties that simply says:

> Only available for columns that are type **numeric**.

I did this because it felt odd to have at the bottom of the second property, when the user might never scroll that far if they only look at the first one. And if they only look at **Min**, they should still be able to find out that it's only available for **numeric** columns.

Anyway, here is the new **Min** and **Max** bits for example:

> ### Min
> Sets the minimum allowed value. For example, you could set the minimum value to 2 if you only want values greater than 2. Any number entered that's less than 2 is considered invalid. Only available for columns that are type **numeric**.

> ### Max
> Sets the maximum allowed value. For example, you could set the maximum value to 100 if you only want values less than 100. Any number entered that's more than 100 is considered invalid. Only available for columns that are type numeric.

Let me know how that feels!